### PR TITLE
Added extra functionality to !xkcd and refactored

### PR DESCRIPTION
**Pull request by tofu-tigre**

### CODE REVIEW

**Comment by robertquitt** [comment]
This is a lot of code to add to bot.py, maybe we could have a separate file for these functions?

**Comment by tofu-tigre** [comment]
@robertquitt I totally understand. Would adding an additional file to apps/ be ok then?

File: apps/discordbot/bot.py [hunk 1]
```diff
@@ -125,31 +129,92 @@ def check_thumb(react, _):
...
+    Assumes that a valid command is contained within msg (checked by is_valid_xkcd_command(String)).
+    """
+    msg = message.content.lower().split();
+    cmd = msg[1]
+    if cmd == VALID_XKCD_COMMANDS[0] or cmd == VALID_XKCD_COMMANDS[1]:
+        await get_random_xkcd(message)
+    elif cmd == VALID_XKCD_COMMANDS[2] or cmd == VALID_XKCD_COMMANDS[3]:
+        await message.channel.send(XKCD_HELP_MSG)
+    elif cmd == VALID_XKCD_COMMANDS[4] or cmd == VALID_XKCD_COMMANDS[5]:
+        await get_xkcd_issue(message, msg[2])
```
Comment (robertquitt) [inline apps/discordbot/bot.py:160]: This is hard to read. Just use the string literals or structure `VALID_XKCD_COMMANDS` differently

File: apps/discordbot/bot.py [hunk 2]
```diff
@@ -22,6 +22,11 @@
 HOSER_ROLE_ID = config("TEST_ROLE", default=785418569412116513, cast=int)  # Verified
 DEBUG_CHANNEL_ID = config("DEBUG_CHANNEL", default=788989977794707456, cast=int)
 TIMEOUT_SECS = 10
+VALID_XKCD_COMMANDS = ["-random", "-r", "-help", "-h", "-issue", "-i"]
+XKCD_RANDOM_URL = "https://c.xkcd.com/random/comic/"
+XKCD_HELP_MSG = "```!xkcd help\n" + ("-" * 50) + "\n-help (-h)  /  Displays help for the '!xkcd' command.\n" \
+                "-random (-r)  /  Displays a random XKCD issue.\n-issue (-i) #  /  Displays a specific XKCD issue.```"
```
Comment (robertquitt) [inline apps/discordbot/bot.py:28]: Use a multi-line string here

```suggestion
XKCD_HELP_MSG = f"""```!xkcd help
{"-" * 50}
-help (-h)  /  Displays help for the '!xkcd' command.
-random (-r)  /  Displays a random XKCD issue.
-issue (-i) #  /  Displays a specific XKCD issue.```
```

**Comment by robertquitt** [comment]
> @robertquitt I totally understand. Would adding an additional file to apps/ be ok then?

Sure, apps/discordbot/xkcd.py would work fine, then just import the functions and call them from bot.py

**Pull request closed without merging**